java/arcs/android/devtools/devtools_message.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package arcs.android.devtools;
+
+import "java/arcs/android/storage/proxy_message.proto";
+import "google/protobuf/wrappers.proto";
+
+option java_package = "arcs.android.devtools";
+
+message DevToolsMessageProto {
+  // ID is a nullable int32, so encode it as an Int32Value.
+  google.protobuf.Int32Value id = 1;
```

Comment:
It doesn't look like you use this, is it actually needed?

If not, you could delete it, and then you can get rid of all that wrappers boilerplate

Reply:
Oops, I changed the structure late Friday and forgot to update the code. I've removed the id and added in the StorageKey and StoreType.